tests/kernels/test_prefix_prefill.py

```diff
@@ -462,3 +476,53 @@ def _get_alibi_slopes(total_num_heads: int) -> torch.Tensor:
     print(f"xformers Time: {(end_time - start_time)*1000:.2f} ms")
     atol = 1e-3 if "fp8" in kv_cache_dtype else 1e-6
     torch.testing.assert_close(output, output_ref, atol=atol, rtol=0)
+
+
+# These tests are optional to only run when explicitly invoked
+#
+# pytest -v -s --optional \
+# tests/kernels/test_prefix_prefill.py::test_contexted_kv_attention_f32
+#
+# These tests are useful to test model dtype float32 on Turing devices.
+# We skip them to not increase the time when running tests on CI
```

Comment:
Just skip for SM 75 won't help because I still need tests to run with dtype float32, which does not run by default. That's why I added this optional mark, it helped me to check that my change is correct without adding extra tests to run on the CI just for this corner case.